pkg/oc/bootstrap/clusteradd/components/automation-service-broker/automation_service_broker.go

```diff
@@ -0,0 +1,39 @@
+package automation_service_broker
+
+import (
+	"github.com/golang/glog"
+
+	"github.com/openshift/origin/pkg/oc/bootstrap"
+	"github.com/openshift/origin/pkg/oc/bootstrap/clusteradd/componentinstall"
+	"github.com/openshift/origin/pkg/oc/bootstrap/docker/dockerhelper"
+)
+
+const (
+	asbNamespace = "automation-broker-apb"
+)
+
+type AutomationServiceBrokerComponentOptions struct {
+	InstallContext componentinstall.Context
+}
+
+func (c *AutomationServiceBrokerComponentOptions) Name() string {
+	return "automation-service-broker"
+}
+
+func (c *AutomationServiceBrokerComponentOptions) Install(dockerClient dockerhelper.Interface, logdir string) error {
+	params := map[string]string{
+		"NAMESPACE": asbNamespace,
+	}
+	glog.V(2).Infof("instantiating automation service broker template with parameters %v", params)
+
+	component := componentinstall.Template{
```

Comment:
I think we're going to require the `WaitCondition` value.

@mfojtik probably worth updating to enforce.  I can't think of a reason not to.